client/profile-wizard/steps/store-details.js

```diff
@@ -26,10 +35,11 @@ import {
 } from 'dashboard/components/settings/general/store-address';
 import UsageModal from './usage-modal';
 import { CurrencyContext } from 'lib/currency-context';
+import { recordEvent } from 'lib/tracks';
 
 class StoreDetails extends Component {
 	constructor( props ) {
-		super( ...arguments );
+		super( props );
```

Comment:
this was just a formatting change, really no need to pass anything but props to `Component`.